pages/Magnesium-Dust.md

```diff
@@ -0,0 +1,13 @@
+Magnesium Dust is one of the nine [dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) added in Slimefun.
```

Comment:
One of these is not like the other

Reply:
I suppose this was related to the "Slimefun" word usage in other places and _this_ sentence is actually correct, right?